From 1dda7b6768e712a8660dfedbf5bafeaea1df5750 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Mon, 08 Mar 2021 11:13:48 +0200
Subject: [PATCH] [SW-20154] gaudi2_arc_gnu: change iatu config

IATU config should include the PCIE Bar size
which is 256MB (28 bits). Hence the correct value
to be programmed into Lower Target Address will
be 0xF000_0000. Higher offset will continue to
be the same as old value.

Change-Id: Ibdd93eca2150173b1990d233d35fd12abcbf1520
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arc_gnu/common.c b/gaudi2_arc_gnu/common.c
index c0cbb95..8d4339c 100644
--- a/gaudi2_arc_gnu/common.c
+++ b/gaudi2_arc_gnu/common.c
@@ -30,7 +30,7 @@
 #define IATU_LWR_TARGET_ADDR_OFF_INBOUND_OFF	0x114
 #define IATU_UPPER_TARGET_ADDR_OFF_INBOUND_OFF	0x118
 
-#define IATU_LOWER_TARGET_ADDR			0xF4000000
+#define IATU_LOWER_TARGET_ADDR			0xF0000000
 #define IATU_HIGHER_TARGET_ADDR			0x1000007F
 #define IATU_INBOUND_LOW_OFFSET_ADDR		0x0
 #define IATU_INBOUND_HI_OFFSET_ADDR		0xC0080000
